### ISSUE TITLE
hide/disable entries in the web interface

### ISSUE THREAD

**Issue by Batwam** [bassmanitram/actions-for-nautilus#57]
Hello,

I love this utility but I was wondering if it would be possible to hide/disable entries in the web interface so they can be kept as template or for future use without being deleted. 

For instance, there could be a checkbox somewhere to disable it which would grey out the text and remove them from the export but leave them in the list?

**Comment by kb-ig** [comment]
Came here for the same thing. I like having the default actions present in the config UI to refer to, but I dont actually need them in the Nautilus context menu. Being able to disable them would be a great addition.

**Comment by bassmanitram** [comment]
Hey there - you mean the delivered actions on first start? I think I could

a) put them in a submenu and
b) add a "disable" flag to submenus and items so that they stay in the
config but don't get parsed or displayed.

then you could disable that submenu when you start adding your own - and
disable any other that you might want to keep around but not use

Would that fit your requirement?

Thanks for using A4N

Martin

**Comment by kb-ig** [comment]
Hi @bassmanitram, thanks for the quick response. 

>you mean the delivered actions on first start

Yeah, I think that's what I'm referring to (the actions that seem to be included by default) - screenshot included below to clarify. 

I dont think it's necessary to hide them all within a sub-menu. In fact, I think doing so would make it more difficult to get familiar with this app. It's useful to have top-level examples of a `command` (e.g. the "Open in Gedit" command).

I think all we need is a bool/switch on each action in the web UI so we can enable/disable them from the Nautilus context menu. If an action is disabled, it's still visible in the web UI but will _not_ appear in the Nautilus context menu. If it's enabled, it's visible in the UI _and_ Nautilus context menu. 

Looking back over the requirement @Batwam described:
>hide/disable entries in the web interface

I think they mean something along the lines of what I'm describing, as opposed preventing the action from appearing in the web UI.

<details>
<summary>Screenshot for reference</summary>

These are the actions that seem to be created by default. They're useful for reference, so I'd like to include them in the UI but _not_ show them in the Nautilus context menu. 

![image](https://github.com/user-attachments/assets/953c8db6-c0be-42ba-9eba-463bce7c019e)

</details>

**Comment by Batwam** [comment]
Yeah, what I meant would be to have entries which are listed in the UI so they can stay visible (eg templates) but do not appear in Nautilus. 

I haven't looked at how the underlying config file looks like but essentially something similar to commenting out a line of code where it's visible to the person setting it up but not actually actioned.

**Comment by bassmanitram** [comment]
Coming in the next release - thank you both for the suggestions
